Branches - Dora

```diff
@@ -0,0 +1,126 @@
+# Create data structure for Driver_ID, Date, Cost, Rider_ID and rating
+drivers = {
+  DR0001: [
+    {
+      date: "3rd Feb 2016",
+      rider_ID: "RD0003",
+      cost: 10,
+      rating: 3,
+    },
+    {
+      date: "3rd Feb 2016",
+      rider_ID: "RD0015",
+      cost: 30,
+      rating: 4,
+    },
+    {
+      date: "5rd Feb 2016",
+      rider_ID: "RD0003",
+      cost: 45,
+      rating: 2,
+    }
+  ],
+  DR0002: [
+    {
+      date: "3rd Feb 2016",
+      rider_ID: "RD0073",
+      cost: 25,
+      rating: 5,
+    },
+    {
+      date: "3rd Feb 2016",
+      rider_ID: "RD0013",
+      cost: 15,
+      rating: 1,
+    },
+    {
+      date: "3rd Feb 2016",
+      rider_ID: "RD0066",
+      cost: 35,
+      rating: 3,
+    }
+  ],
+  DR0003: [
+    {
+      date: 20160204,
+      rider_ID: "RD0066",
+      cost: 5,
+      rating: 5,
+    },
+    {
+      date: 20160205,
+      rider_ID: "RD0003",
+      cost: 50,
+      rating: 2,
+    }
+  ],
+  DR0004: [
+    {
+      date: 20160203,
+      rider_ID: "RD0022",
+      cost: 5,
+      rating: 5,
+    },
+    {
+      date: 20160204,
+      rider_ID: "RD0022",
+      cost: 10,
+      rating: 4,
+    },
+    {
+      date: 20160205,
+      rider_ID: "RD0073",
+      cost: 20,
+      rating: 5,
+    }
+  ]
+}
+
+
+earnings = {}  # Hash to store driver as key and cost as values
+avg_rating = {}  # Has to store driver as key and average rating as value
+
+drivers.each do |driver, array|
+  # The number of rides each driver has given
+  puts "Driver #{driver} drove a total of #{array.count} rides."
+  earnings[driver] = []
+  avg_rating[driver] = []
+  array.each do |trip|
+    earnings[driver] << trip[:cost]
+    avg_rating[driver] << trip[:rating]
```

Comment:
Wanted to call out that you're calculating the average later on and this array would be more accurately named ratings.